src/frontend/src/components/SearchResults/Case.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Why are this Component and Cases in the SearchResults directory instead of their own directory? I don't think we do this anywhere else in the app right now. Is there a good reason to deviate from convention here?

Reply:
fetch has not been named fetchRecords for clarification.  Also, Cases and Case components have been moved to their own directory.